Merchant profile and payment method updates (#374)
swan-io/docs: docs/topics/merchants/index.mdx

Subscribe to numerous webhooks to facilitate your merchant integrations.
Review the [full list of merchant webhooks](../../developers/using-api/webhooks.mdx#events-merchants), or subscribe to them directly from your **Dashboard** > **Developers** > **Webhooks**.

## Merchant profiles {#profiles}

A **merchant profile** is a collection of business, risk, and support information that fulfills two primary requirements: 

> 1. **Approve the account holder** as a merchant.

Author: In our documentation, we use "Account Holder" to refer to the **entity that owns the account** and "User"  as a general term for the **individual interacting with the system**. 

I updated the text to make some swaps where needed:

- I used **Account holders** where we're referring to the entity that qualifies and is setting up to receive payments. 

- I kept **Users** where we're referring to the individual interacting with the system on behalf of the account holder.

1. **Facilitate accepting payments** with Swan. 

If a Swan account holder wants to accept payments from customers, they must have at least one merchant profile.

Your users can create merchant profiles during onboarding, but only if you've requested the feature for them.

> Since merchants are considered high risk, merchant profiles should only be requested for users who qualify and will accept payments.

Author: We mention that Swan reviews the submission, but we don’t specify the team or criteria. This is intentional to keep the docs clear and avoid exposing sensitive internal processes.

Requesting merchant profiles for users that don't qualify may create false expectations about available features.

A merchant profile contains the following information:

| Field | Description |
| --- | --- |
| **Business details** | Required. The merchant's name, product type (`product`, `services`, `digital goods`, or `donations`), and a short description of their activity. |

> | **Projected volumes** | Required. Include the expected monthly payment volume and average order value. Helps assess business size and risk. |

Reviewer: ```suggestion
| **Projected volumes** | Required. Includes the expected monthly payment volume and average order value. This helps us assess the business size and risk. |
```

> | **Link to the merchant's website or social network** | Highly recommended. This helps us understand the merchant's business. Sometimes required. |

Reviewer: Why is this highly recommended but sometimes required? These two bits of information should be together.

> | **Description of the business activity** | Highly recommended. A clear explanation of what the merchant offers or sells. |

Reviewer: So we have description of business activity here, but also in the Business details section above? Are we asking them the same information twice?

> | **Payment frequency** | Specify if payments are mostly one-off or recurring (e.g. subscriptions). |

Reviewer: The descriptions should either be an explainer or an action - at the moment they're a mix of both. 

If it's an explainer - change this to "Defines whether payments are one-off or recurring"

| **Logo** | Optional. Must be a `.png` file under 1 MB. Displayed online and on the mandate PDF. |
| **Accent color** | Optional. Used to customize the mandate and payment page design. |

> | **Customer region** | The country or region from which the merchant expects to receive payments. |

Reviewer: Required or optional?

| **Additional information** | Required only if custom fields are needed. Details are shared in the `expected` section of the project. |

> | **Supporting documents** | Required only after a risk review. Specific documents are listed in the `paymentMethod` object via the API. |

Reviewer: ```suggestion
| **Supporting documents** | Required only after a risk review. Specific documents are listed in the `paymentMethod` object using the API. |
```

> :::tip 

Reviewer: I'd move this tip up - or have it as a sentence in the intro to the table.

To speed up **merchant profile enablement**, complete as many input fields as possible.  

> Providing your `customerRegion`, `paymentFrequency`, and at least a `socialNetwork` or `merchantWebsite` can reduce turnaround time.

Reviewer: Unsure why we need this when we have the required/optional/highly recommended in the table above.

        D
        R
    end
    style MPR fill:#D4D3D5,stroke:#514F59
    style F fill:#fffdf4,stroke:#cca922
    style U fill:#dfedf2,stroke:#4d8296, stroke-width:3px
```

| Profile status | Explanation |
|---|---|

> | `PendingReview` | The profile request or update was submitted and is pending a review by Swan. <br /><br />• If the request doesn't meet requirements or is deemed risky, the status moves directly to `Rejected`.<br />• Otherwise, the status moves to `Enabled`. <br /><br />**Next steps:**<br />• Use `PendingReview` merchant profiles to request payment methods. |

Reviewer: Suggest simplifying: 
```suggestion
| `PendingReview` | The profile is under review by Swan. <br /><br />• If the profile doesn't meet our requirements or we consider it a risk, we'll change the status to `Rejected`.<br />• Otherwise, we'll change the status to `Enabled`. <br /><br />Capabilities:<br />• While a profile is PendingReview, you can still request payment methods. |
```

> | `Enabled` | The profile is active.<br /><br />• Use `Enabled` profiles to request payment methods and accept payments with activated payment methods.<br />• Updates to profiles can only occur when they are `Enabled`. |

Reviewer: Can we simplify this. 
```suggestion
| `Enabled` | The profile is active. <br /><br />Capabilities:<br />• Request payment methods and accept payments.<br />• Make updates to the profile's information.|
```

> | `Suspended` | Status **assigned by Swan** when the merchant's use of their profile needs to be reviewed. The status can't return to `Enabled` without a review. |

Reviewer: ```suggestion
| `Suspended` | **Swan assigns this status** when we need to review a merchant's profile. The profile can't be `Enabled` again until we complete our review. |
```

Reviewer: More active.

> | `Rejected` | Final status **assigned by Swan** when the risk of this profile is too high, or for another risk-related reason. |

Reviewer: ```suggestion
| `Rejected` | Swan assigns this final status when a profile's risk is too high, or for other risk-related reasons. |
```

> | `Disabled` | Final status **set by you** using the `disableMerchantProfile` mutation. |

Reviewer: ```suggestion
| `Disabled` | You set this final status using the `disableMerchantProfile` mutation. |
```

All update requests are reviewed by Swan according to the same [risk guidelines](#risk) as a new merchant profile.
:::

### Merchant profile rejection {#profile-rejected}

The merchant profile is created with the status PendingReview, and is visible right away on your Dashboard. 

> It is then updated to either Enabled or Rejected depending on the outcome of Swan's risk review.

Reviewer: Should this be: 
```suggestion
It is then updated to either `Enabled` or `Rejected` depending on Swan's risk review.
```

A merchant profile can be rejected for the following reasons:

| Rejection reason | Explanation |
| --- | --- |

> | `DuplicateProfile` | The profile is a duplicate of an existing one. |

Reviewer: ```suggestion
| `DuplicateProfile` | The profile is the same as an existing one. |
```

| `InconsistentOrMissingInformation` | Key information is missing or highly inconsistent—for example, an unrelated website. |
| `UnsupportedActivity` | The activity is prohibited or the NACE code isn't supported. |

> | `ContractualCriteriaNotFulfilled` | The profile doesn't meet specific criteria agreed upon with partners. |

Reviewer: ```suggestion
| `ContractualCriteriaNotFulfilled` | The profile doesn't meet specific criteria agreed upon with Partners. |
```

| `GenericRejection` | The profile was rejected for a general reason not covered by other categories. |
| `SuspiciousBehavior` | The profile was rejected due to alerts related to suspicious processing behavior. |

## Payment methods {#methods}

> Your merchants can accept payments from their customers using any of several **merchant payment methods**.

Reviewer: ```suggestion
Your merchants can accept customer payments using several payment methods.
```
Simplify.

<OnlinePaymentMethods />

All merchant payment method requests are subject to a risk review and start with the status `PendingReview`. 
As part of this process, Swan may need to request additional information from the merchant to complete the assessment.

### Payment method statuses {#method-statuses}

```mermaid
%%{ init: { 'flowchart': { 'curve': 'linear' } } }%%

flowchart LR

PMQ((Payment method<br/>requested)) --> PR[PendingReview]
PR <-.-> WFI[WaitingForInformation]
PR --> R[Rejected]
PR --> E[Enabled]
E <-.-> S[Suspended]
S --> R
S --> D[Disabled]
E --> D

subgraph U[Can use]
    E
end

subgraph F[Final status]
    D
    R
end

style PMQ fill:#D4D3D5,stroke:#514F59
style F fill:#fffdf4,stroke:#cca922
style U fill:#dfedf2,stroke:#4d8296, stroke-width:3px
```

| Payment method status | Explanation |
|---|---|

> | `PendingReview` | The payment method request or update was submitted and is awaiting a review by Swan.<br /><br />• If the request doesn't meet requirements or is deemed risky, the status moves directly to `Rejected`.<br />• Otherwise, the status moves to `Enabled`.|

Reviewer: Is this the same as the statuses above?

Reviewer: If yes, these can all be simplified.

| `Enabled` | The payment method is active.<br /><br />• Use `Enabled` payment methods to accept payments.<br />• Updates to payment methods can only occur when they are `Enabled`. |
| `Suspended` | Status **assigned by Swan** when the merchant's use of their payment method needs to be reviewed. The status can't return to `Enabled` without a review. |
| `Rejected` | Final status **assigned by Swan** when the risk of this payment method is too high, or for another risk-related reason. |
| `Disabled` | Final status **set by you** using the `disableMerchantPaymentMethod` mutation. |

### Payment method rejection {#payment-method-rejected}

A merchant payment method can be rejected for the following reasons:

| Rejection reason | Explanation |
| :--- | :--- |

> | `InvalidUseCase` | The requested payment method doesn't make sense for the merchant's business model. |

Reviewer: This is a mix of formal and informal tone - try: This payment method is incompatible with the merchant's business model - to stick with formal.

| `InconsistentOrMissingInformation` | The merchant failed to provide, or refused to provide, requested information. |

> | `UnsupportedPaymentMethod` | The partner isn't allowed to request this payment method. |

Reviewer: ```suggestion
| `UnsupportedPaymentMethod` | The Partner isn't allowed to request this payment method. |
```

Reviewer: What is it based on? "The Partner isn't allowed to request this payment method based on...." - so it's clear why they're not allowed.

> | `ContractualCriteriaNotFulfilled` | The request violates a specific criterion agreed upon with the partner. |

Reviewer: ```suggestion
| `ContractualCriteriaNotFulfilled` | The request violates a specific criterion agreed upon with the Partner. |
```
